README.md

```diff
@@ -220,12 +254,14 @@ annotation to your .proto file
    }
    ```
 
-7. (Optional) Generate swagger definitions using `protoc-gen-swagger`
+6. (Optional) Generate swagger definitions using `protoc-gen-swagger`
 
    ```sh
    protoc -I. --swagger_out=logtostderr=true:./gen/swagger your/service/v1/your_service.proto
    ```
```

Comment:
Lets update the rest of the invocations in the README while we're here, I think you can see the pattern.
```suggestion
   ```sh
   protoc -I . --swagger_out ./gen/swagger --swagger_opt logtostderr=true your/service/v1/your_service.proto
   ```
```